website/server/controllers/api-v3/webhook.js

```diff
@@ -179,7 +179,7 @@ api.updateWebhook = {
       webhook.url = url;
     }
 
-    if (label) {
+    if (label !== null && label !== undefined) {
```

Comment:
could you add a comment here to explain that we're using this check to allow the setting of empty labels? Thanks!

Reply:
@paglias added the comment!